LibrariesPrototype/Presentation/Extensions/UIImageView+AlamofireImage.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  UIImageView+AlamofireImage.swift
+//  LibrariesPrototype
+//
+//  Created by Ozgun Zor Personal on 05/05/2019.
+//  Copyright © 2019 Ozgun Zor. All rights reserved.
+//
+
+import AlamofireImage
+import UIKit
+
+extension UIImageView {
```

Comment:
why we need public func ? if necessary, public before extension should be ok. 
https://stackoverflow.com/questions/33925840/access-control-for-swift-extensions

Reply:
In addition, careful using access control with classes, as I understand they are different from extensions. 
for more look at "CUSTOM TYPES" title 
https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html